examples/SC_noise_capture.py

```diff
@@ -0,0 +1,148 @@
+"""
+SpinCore Noise Acquisition
+==========================
+
+This script acquires a set number of captures of the noise
+as seen by the SpinCore. Remember to add a terminated attenuator 
+to the output of the SpinCore prior to measurement!
+"""
+import pyspecdata as ps
+import numpy as np
+import os
+from numpy import r_
+from timeit import default_timer as timer
+import SpinCore_pp as sc
+from datetime import datetime
+
+
+# {{{ Function for data acquisition
+def collect(config_dict, my_exp_type):
+    """
+    Collects a number of captures (specified by the config file) of the noise
+    as acquired on the SpinCore. Each capture is stored along the "nScans"
+    dimension with a direct "t" axis.
+    :Note: Some delays (e.g. the repetition
+    and tau are hard set to short values and are not pulled or stored in the
+    config file).
```

Comment:
could pull from config dict but figured it's more straight forward to just hardset short values